packages/media/src/generateVideoStrip.js

```diff
@@ -112,6 +113,20 @@ async function generateVideoStrip(element, resource, stripWidth, stripHeight) {
     // Seek to the next offset
     await seekVideo(video, timeOffset);
 
+    const dx = frame * actualStripFrameWidth;
+
+    if (flip.vertical || flip.horizontal) {
+      // translate context to center of canvas
```

Comment:
Nit (for the comments below as well):
```suggestion
      // Translate context to the center of the canvas.
```